cypress/e2e/supervision/clients/client-tasks-tab.cy.js

```diff
@@ -35,7 +35,7 @@ describe(
       cy.get('.details-row > td').should('contain.text', 'A client has been created');
 
       cy.get('.order-number').should('contain.text', 'PFA');
-
+      cy.wait(500);
```

Comment:
What is this waiting for? Is it not possible to ensure the page is in the correct state by asserting on an element?

Reply:
To make sure that angular has finished loading everything on the pages

Reply:
Is there an element or class it can wait on instead then?

Reply:
Its the case that it looks like it clicks the button but then other api calls come back and override the button click so never gets to that page it needs.